tools/sponge/preproc_scripts/get_so_monthly.csh

```diff
@@ -0,0 +1,31 @@
+#!/bin/tcsh
+#SBATCH --ntasks=1
+#SBATCH --job-name=fill_glorys_arctic
+#SBATCH --time=2880
+#SBATCH --partition=batch
+
+# Usage: sbatch fill_glorys_nn_monthly.sh <YEAR> <MONTH>
+
+module load cdo
+module load nco/5.0.1
+module load gcp
+
+set year=$1
+set month=`printf "%02d" $2`
+
+set apath='/archive/Theresa.Morrison/datasets/glorys/GLOBAL_MULTIYEAR_PHY_001_030/monthly/so'
+mkdir ${apath}/so_${year}_${month}
+
+set day=1
+foreach filename (/uda/Global_Ocean_Physics_Reanalysis/global/daily/so/${year}/so_mercatorglorys12v1_gl12_mean_${year}${month}*.nc)
+  echo $filename 
+  set short_name='so_arctic_'$day
+  ncks -d latitude,39.,91. --mk_rec_dmn time $filename ${apath}/so_${year}_${month}/${short_name}'_bd.nc'
```

Comment:
Ideally these temporary files would be written to $TMPDIR and only the final result would be copied to /archive

Reply:
Done - only the final file is written to archive.